src/helper/accept/index.ts

```diff
@@ -0,0 +1 @@
+export { matchAccept } from './accept'
```

Comment:
I'm not sure if `matchAccept` is the best name. I think `accepts` simply is better.